Makefile

```diff
@@ -234,46 +240,47 @@ bench-privileged:
 		|| exit 1; \
 	done
 
-clean-tags:
+clean-tags: ## Remove all the cscope tags file from the repository.
 	@$(ECHO_CLEAN) tags
 	@-rm -f cscope.out cscope.in.out cscope.po.out cscope.files tags
 
-cscope.files: $(GOLANG_SRCFILES) $(BPF_SRCFILES)
+cscope.files: $(GOLANG_SRCFILES) $(BPF_SRCFILES) ## Generate cscope.files with the list of all files to generate ctags for.
 	@echo $(GOLANG_SRCFILES) $(BPF_SRCFILES) | sed 's/ /\n/g' | sort > cscope.files
 
-tags: $(GOLANG_SRCFILES) $(BPF_SRCFILES) cscope.files
+tags: $(GOLANG_SRCFILES) $(BPF_SRCFILES) cscope.files ## Generate tags for golang and bpf source files.
 	@ctags $(GOLANG_SRCFILES) $(BPF_SRCFILES)
 	cscope -R -b -q
 
-clean-container:
-	-$(QUIET) for i in $(SUBDIRS); do $(MAKE) $(SUBMAKEOPTS) -C $$i clean; done
+clean-container: ## Perform cleanup for components required in cilium-agent container.
+	-$(QUIET) for i in $(SUBDIRS_CILIUM_CONTAINER); do $(MAKE) $(SUBMAKEOPTS) -C $$i clean; done
 
-clean: clean-container
+clean: ## Perform overall cleanup for Cilium.
+	-$(QUIET) for i in $(SUBDIRS); do $(MAKE) $(SUBMAKEOPTS) -C $$i clean; done
 	-$(QUIET) $(MAKE) $(SUBMAKEOPTS) -C ./contrib/packaging/deb clean
 	-$(QUIET) $(MAKE) $(SUBMAKEOPTS) -C ./contrib/packaging/rpm clean
 
-veryclean:
+ctrclean: ## Perform complete cleanup for container engine images(including build cache).
```

Comment:
Why was this renamed?

Reply:
Had this local change from before, will update the target name.